.github/workflows/tests_linux.yml

```diff
@@ -71,18 +71,31 @@ jobs:
               -DENABLE_COVERAGE=ON \
               -DLQ_ENABLE_KERNEL_OMP=ON
 
+            cmake . -BBuildKernelAVXStream -G Ninja \
```

Comment:
do we want to always check the C++ tests? can we limit this to only pushes to master for example to remove the build and test time on every commit?

Reply:
Let's do this in the other PR that is restricting CPP tests across the board, and get this in first.

Reply:
Let's have it done then when we split cpp tests! :+1: